src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -118,13 +124,9 @@ public String getInternalLinkRedirectTitle(int index) {
     return pInfoInternalLinks.get(index).getAttribute("href");
   }
 
-  public WebElement getItemLabel() {
-    return itemLabel;
-  }
+  public WebElement getItemLabel() { return itemLabel; }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)